components/Formsy/TiledRadioGroup.jsx

```diff
@@ -8,24 +8,37 @@ import { HOC as hoc } from 'formsy-react'
 class TiledRadioGroup extends Component {
   constructor(props) {
     super(props)
+    this.state = {
+      curValue: []
+    }
     this.onChange = this.onChange.bind(this)
+    this.state.curValue = (this.props.getValue())
   }
 
   onChange(value) {
+    const index = this.state.curValue.indexOf(value)
+    if (this.props.multipleOptions) {
+      if (index > -1) {
+        this.state.curValue.splice(index, 1)
+      } else {
+        this.state.curValue.push(value)
+      }
+    } else {
+      this.state.curValue = [value]
+    }
     this.props.setValue(value)
-    this.props.onChange(this.props.name, value)
+    this.props.onChange(this.props.name, this.state.curValue)
```

Comment:
This would break the existing usage - it's always passing the curValue array, but if it's selecting just one value, it should pass just that one value, not an array.

Reply:
@gondzo updated